openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -500,7 +502,7 @@ def process(self, instance):
 
         self.payload_skeleton["JobInfo"]["Comment"] = comment
         self.payload_skeleton["PluginInfo"]["RenderLayer"] = renderlayer
-
+        self.payload_skeleton["PluginInfo"]["RenderSetupIncludeLights"] = get_current_project_settings()["maya"]["create"]["CreateRender"]["disable_all_lights"] # noqa
```

Comment:
Shouldn't this use `context.data["project_settings"]` like [here](https://github.com/pypeclub/OpenPype/blob/8b8269a1324045faf134aafd6d654bb0c4fe9a0d/openpype/modules/deadline/plugins/publish/submit_maya_deadline.py#L332)?

Reply:
I kinda like the explicit one better that just plops it down from that explicit function tbh.

Reply:
We should keep to the settings that have already been collected, rather than calling for them again, that's exactly why we collect them to the context. Keep in mind that all of these are db calls.

Reply:
Oh my god you're absolutely right, fixing that now!

Reply:
Done!